pages/settings/profile.tsx

```diff
@@ -268,27 +277,29 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
                   name="avatar"
                   id="avatar"
                   placeholder="URL"
-                  className="block w-full px-3 py-2 mt-1 border border-gray-300 rounded-sm shadow-sm focus:outline-none focus:ring-neutral-500 focus:border-neutral-500 sm:text-sm"
+                  className="block w-full px-3 py-2 mt-1 border border-gray-300 rounded-sm shadow-sm focus:outline-none focus:ring-neutral-800 focus:border-neutral-800 sm:text-sm"
                   defaultValue={imageSrc}
                 />
-                <ImageUploader
-                  target="avatar"
-                  id="avatar-upload"
-                  buttonMsg={t("change_avatar")}
-                  handleAvatarChange={(newAvatar) => {
-                    avatarRef.current.value = newAvatar;
-                    const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
-                      window.HTMLInputElement.prototype,
-                      "value"
-                    )?.set;
-                    nativeInputValueSetter?.call(avatarRef.current, newAvatar);
-                    const ev2 = new Event("input", { bubbles: true });
-                    avatarRef.current.dispatchEvent(ev2);
-                    updateProfileHandler(ev2 as unknown as FormEvent<HTMLFormElement>);
-                    setImageSrc(newAvatar);
-                  }}
-                  imageSrc={imageSrc}
-                />
+                <div className="flex items-center px-5">
+                  <ImageUploader
+                    target="avatar"
+                    id="avatar-upload"
+                    buttonMsg={t("change_avatar")}
+                    handleAvatarChange={(newAvatar) => {
+                      avatarRef.current.value = newAvatar;
+                      const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
+                        window.HTMLInputElement.prototype,
+                        "value"
+                      )?.set;
+                      nativeInputValueSetter?.call(avatarRef.current, newAvatar);
+                      const ev2 = new Event("input", { bubbles: true });
+                      avatarRef.current.dispatchEvent(ev2);
+                      updateProfileHandler(ev2 as unknown as FormEvent<HTMLFormElement>);
+                      setImageSrc(newAvatar);
+                    }}
+                    imageSrc={imageSrc}
+                  />
+                </div>
```

Comment:
Not part of teams, but I changed the form focus colors to match and adjusted the ImageUploader to reflect changes within that component.